src/modules.py

```diff
@@ -132,7 +132,7 @@ def parse(config: Dict[str, Any], iocs: IocCollection) -> None:
 
     for name in ["email", "emails"]:
```

Comment:
Do you think it's a good idea to add support for `email_from` and `email_to` fields? But maybe it's unnecessary future thinking. Currently only agenttesla uses that.